ush/nems_configure.sh

```diff
@@ -155,10 +155,10 @@ EOF
 
 echo "$(cat med_modelio.nml)"
 
-cp $HOMEgfs/sorc/ufs_coupled.fd/CMEPS-interface/CMEPS/mediator/fd_nems.yaml fd_nems.yaml
-
 fi 
 
+cp $HOMEgfs/sorc/ufs_coupled.fd/tests/parm/fd_nems.yaml fd_nems.yaml
```

Comment:
This is an interesting question.

Yes, the NUOPC field dictionary is not required by standalone atm. Furthermore, recent FV3 versions no longer rely on the `cpl` flag, formerly in the `model_configure` file, to advertise/realize coupling fields in the NUOPC cap. We could wrap this file copy in a conditional block to check whether any of the physics coupling flags (`cplflx`, `cplwav`, etc.) are true, similarly to what implemented in `cplvalidate.sh`:
https://github.com/NOAA-EMC/global-workflow/blob/3428097dd39934ea648d86e5cdee8d8c2bdb3b14/ush/cplvalidate.sh#L32
However, there is an ongoing effort to refactor the FV3 cap, with the possible (but remote) outcome that other coupling flags may no longer be used, too.

Reply:
This isn't just used by the atm model in coupling. It's used by any model.  Did the "cpl" flag get removed completely? I believe it's still a variable in the workflow if not in input.nml/model_configure 

At any rate,  it doesn't adversely affect fv3 standalone so it can just be left as is for simplicity.

Reply:
Yes, the `cpl` flag was removed and it is no longer needed in `model_configure`.